tests/integration/features/notifications.feature

```diff
@@ -0,0 +1,21 @@
+Feature: notifications
+  Background:
+    Given using api version "1"
+
+  Scenario: Create test user
```

Comment:
scenarios are to be self contained - "Creating a new notification" relies on "Create test user"

further more "Create test user" has nothing to do with notifications.

That a given user exists is a precondition to the scenario - this is why the original implementation did use

Given user "user01" exists